### ISSUE TITLE
Add options to configure button position 

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-scrollPrompt#12]
As per [#4 PR](https://github.com/cgkineo/adapt-scrollPrompt/pull/4#issuecomment-1601313937), add optional config for button positioning.

`.scrollPrompt__inner` already uses `display: flex`. Using the flex properties would also be consistent with how we already handle configurable layouts in [Vanilla](https://github.com/adaptlearning/adapt-contrib-vanilla/pull/364).

**Comment by swashbuck** [comment]
I agree that this would be useful. In a project I'm currently working on, we stack the button and text vertically.

<img width="285" alt="Screenshot 2023-08-14 at 12 58 14 PM" src="https://github.com/cgkineo/adapt-scrollPrompt/assets/898168/a38589ce-5ca1-4221-b7f7-3c99f987318c">
